### PR TITLE
Storage

### PR DESCRIPTION
**Pull request by christopher77**
Add cover to the models:
-Songs
-Albums
-Artists

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -15,6 +15,27 @@
...
+  create_table "active_storage_attachments", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "record_type", null: false
+    t.bigint "record_id", null: false
+    t.bigint "blob_id", null: false
+    t.datetime "created_at", null: false
+    t.index ["blob_id"], name: "index_active_storage_attachments_on_blob_id"
+    t.index ["record_type", "record_id", "name", "blob_id"], name: "index_active_storage_attachments_uniqueness", unique: true
+  end
+
```
Comment (CesarCachay) [inline db/schema.rb]: Those are the tables, created when we install activestorage

File: app/views/admin/albums/show.html.erb [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <h2>Album <%= @album.title %></h2>
 <div>
   <p><b>Rating:</b> <%= @album.rating %></p>
+  <% if @album.cover.attached? %>
+    <%= image_tag @album.cover.variant(resize: "300x300") %>
```
Comment (CesarCachay) [inline app/views/admin/albums/show.html.erb]: Nice we are including mini-magick to transform the size of the picture.

Comment (christopher77) [reply app/views/admin/albums/show.html.erb]: yes, It is already fixed

File: Gemfile [hunk 3]
```diff
@@ -28,7 +28,7 @@ gem 'jbuilder', '~> 2.5'
 # gem 'bcrypt', '~> 3.1.7'
 
 # Use ActiveStorage variant
-# gem 'mini_magick', '~> 4.8'
+gem 'mini_magick', '~> 4.8'
```
Comment (AngieCristina) [inline Gemfile]: Good election, to use this gem in your project.

**Pull request merged**